cmd/init.go

```diff
@@ -0,0 +1,120 @@
+package cmd
```

Comment:
Oh, can we also encapsulate this in its own package?  So the package here would be `package init`.

And let's move `init.go` and `init_test.go` into `cmd/init/cmd.go` etc.  This might not make sense with just the one command here, but if this ever grows to have lots of commands it ends up being a bit more easy to navigate.